lighthouse-cli/test/check-formatting.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+diff -u <(cat *.ts) <(./node_modules/.bin/clang-format -style=file *.ts)
+if [ $? -ne 0 ]; then
```

Comment:
i'd recommend throwing your script at https://www.shellcheck.net/
it already has a few suggestions i'd take.